RELEASING.md

```diff
@@ -4,32 +4,22 @@ Steps to releasing a new version of DeText involves the following steps: 1) publ
 ## Publishing to PyPi
 NOTE: this guide is for DeText owners to publish new versions to PyPi
 
-1. Make sure you have the correct permission to release DeText packages. Only owners and maintainers can upload new releases for DeText. Check more details at https://pypi.org/project/detext/.
-
-2. Create a source distribution by:
-* update the version in `setup.py`
-* comment the following lines (for internal use, we cannot have these dependencies in the pypi package)
-```
-        'tensorflow==1.14.0',
-        'tensorflow_ranking==0.1.4',
-        'gast==0.2.2'
-```
-Then run the following:
-```
-python setup.py sdist
-```
-
-3. Install `twine` for uploading to PyPi
-
-```
-pip install twine
-```
-
-4. Upload the distribution
-
-```
-twine upload dist/*
-```
+ Make sure you have the correct permission to release DeText packages. Only owners and maintainers can upload new releases for DeText. Check more details at https://pypi.org/project/detext/.
+
+### Steps:
+1. Install `twine` for uploading to PyPi
+    ```shell script
+    pip install twine
+    ```
+1. Increment `VERSION` in `setup.py`. You can checkout previous versions at https://pypi.org/manage/project/detext/releases/ and determine the new release version. 
```

Comment:
You can also consider update the version automatically. The latest release version available from pypi-cli, such as `pypi info detext`. More details: https://pypi.org/project/pypi-cli/.

Reply:
Thanks! That seems like a great tool. We can use it to check the latest version. But will still need to manually increment the versions in case we need to bump major/minor versions. 
I'm looking at some versioning tool like this one https://github.com/c4urself/bump2version/#installation 
Will integrate it for automatic version bump